### DISCUSSION TITLE
主页市场 v2

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [Meloong-Git/PCL#4668] [自定义主页]
全新的构建器，全新的主页市场，由 @MFn233 和我共同创建！
部分样式资料引用自 Minecraft 新闻主页

GitHub 仓库：[HomePlaza-Of-PCL2/Homepage-Market](https://github.com/HomePlaza-Of-PCL2/Homepage-Market)
反馈：[New Issue](https://github.com/HomePlaza-Of-PCL2/Homepage-Market/issues/new/choose)
新主页投稿：[这里](https://github.com/HomePlaza-Of-PCL2/Homepage-Market/issues/new?assignees=&labels=a%3A%E6%96%B0%E4%B8%BB%E9%A1%B5&projects=&template=4_new.yml)
加载地址：[这里](https://homepage-market.pages.dev/Custom.xaml)

TODO：
- [x] INI 版本文件
- ~~更换加载地址（现在这个地址貌似有概率报错）~~
- [x] UI 看看再美化美化
- [ ] Web 预览站点 <不是主页相关，预期10月上线>

<details>
<summary>截图预览</summary>

最小窗口
![image](https://github.com/user-attachments/assets/a6c4d9aa-cef5-46ef-9918-59963567872f)
最大窗口（屏幕 2160x1440）
![image](https://github.com/user-attachments/assets/bf6168de-5142-4389-ad76-6f81def243ba)

</details>

**Comment by wkea** [comment]
很棒 建议收录

**Comment by MFn233** [comment]
很棒 建议收录
@LTCatt

**Comment by LTCatt** [comment]
做完之后艾特我一下，我加进去（

**Reply by MoYuan-CN** [reply]
@LTCatt
好了，地址不变（鬼知道之前为啥抽风了）
对部分 UI 做了调整

**Reply by LTCatt** [reply]
https://raw.gitcode.com/HomePlaza-Of-PCL2/Homepage-Market/raw/main/Custom.xaml
目前 404

**Reply by MoYuan-CN** [reply]
我看着是好的？

**Reply by 3gf8jv4dv** [reply]
> 我看着是好的？

https://gitcode.com/org/HomePlaza-Of-PCL2/repos

I cannot see any repos. Is it set to private?

**Reply by MoYuan-CN** [reply]
Private？
![image](https://github.com/user-attachments/assets/2014f10a-3110-45a6-87ee-05eea61d91df)

**Reply by LTCatt** [reply]
![image](https://github.com/user-attachments/assets/f13349b6-c5d5-4f79-b18e-4d47f7462ec7)

**Reply by MoYuan-CN** [reply]
Gitcode 等国内服务的坏坑：得等审核
等我换个地址好了

**Comment by MoYuan-CN** [comment]
主文件 https://homepage-market.pages.dev/Custom.xaml
检查地址 https://homepage-market.pages.dev/Custom.xaml.ini 
@LTCatt

**Reply by MoYuan-CN** [reply]
要是还不行就 Draft 掉计划，等啥时候 Gitcode 能用了再说

**Reply by LTCatt** [reply]
如果用了信标的部分模板，记得加一个 credit（
新版本塞进去了 :D

**Reply by MoYuan-CN** [reply]
好，加上了

**Comment by MoYuan-CN** [comment]
实际上的 Web 主页已经鸽到不知道什么时候去了，如果有朋友期待的话可以先放一放了……

**Reply by MoYuan-CN** [reply]
@LTCatt 麻烦切个地址 https://pclhomeplazaoss.lingyunawa.top:26994/d/Homepages/JingHai-Lingyun/Custom.xaml
之前的地址会持续提供服务直到切流结束后 14 个自然日

**Reply by LTCatt** [reply]
将在下个版本生效

**Comment by MoYuan-CN** [comment]
哈喽
时隔多年的一次更新
地址变更为了 https://pclhomeplazaoss.lingyunawa.top:26994/p/Homepages/Homepage.Market/Custom.xaml

旧链接保留到 2026.7.1
这次更新是为了响应 HomeplazaOSS 语义化路径的新标准，所以可能你过一阵子还会收到几个更新

**Reply by MoYuan-CN** [reply]
啊 忘了 @LTCatt :(

**Comment by CreeperIsASpy** [comment]
题外话：新的构建器 UHML.Builder 非常棒，建议各位主页制作者去体验一下！
https://github.com/HomePlaza-Of-PCL2/UHML.Builder
